content/pages/irc.md

```diff
@@ -29,42 +29,81 @@ systems committee channel,
 [#wics-sys](http://webchat.oftc.net/?channels=wics-sys), in this way. Simply
 choose a username and click "Connect"!
 
-## Graphical clients ##
-
-Follow these instructions on the OpenHatch wiki to download and install an IRC
-client:
-
-+ [Windows](https://openhatch.org/wiki/Open_Source_Comes_to_Campus/Curriculum/Laptop_setup/Windows_IRC)
-+ [OS X](https://openhatch.org/wiki/Open_Source_Comes_to_Campus/Curriculum/Laptop_setup/OSX_IRC)
-+ [Linux](https://openhatch.org/wiki/Open_Source_Comes_to_Campus/Curriculum/Laptop_setup/Linux_IRC)
-
-Our connection information differs from the OpenHatch wiki. Please use the
-following parameters:
-
-**Server name:** OFTC (Open and Free Technology Community)<br>
-**Channel name:** #wics
-
-## Persistent sessions ##
-
-Many of our users have an IRC session online at all times. If you'd like to
-use a similar setup, you will need access to a Linux server.
-
-[IRCCloud](https://www.irccloud.com/) is a user-friendly and feature rich
-graphical client that supports persistent sessions, file and image sharing, and
-more through web and mobile clients. However, it costs $5 USD/month for
-persistent sessions. If you're looking for a more modern take on IRC that
-includes a mobile application with out-of-the-box push notifications and you're
-not worried about the small cost, this may be a great choice for you.
-
-For a more traditional setup, you can [follow these
-instructions](http://wiki.csclub.uwaterloo.ca/How_to_IRC) to configure IRC in a
-screen session on a remote server, allowing you to use a secure shell to log in
-and view your session at any time.
-
-The instructions above are aimed at Computer Science Club members, who have
-login credentials for `taurine.csclub.uwaterloo.ca`. Unfortunately, at this
-time, WiCS does not offer community logins. We hope to have a server set up in
-the near future to provide this service. In the meantime, University of
-Waterloo students, staff, and alumni can purchase a
-[Computer Science Club](https://csclub.uwaterloo.ca) membership for $2 per
-term, which grants you access to the CSC network.
+#### WeeChat
+Weechat has become a favourite among committee, so if you like to have a gui
+and point-and-clickness, we recommend you try this first. It also has
+notifications on phone and desktop, and you are always connected if you keep it
+on a screen.
+
+Steps:
+
+1. ssh into taurine or mother gooose, by running one of these commands in PuTTY
+   (Windows) or in a Terminal (Mac/Linux):
+   - `ssh userid@taurine.csclub.uwaterloo.ca` (for CSC) - If you don’t have an
+      account, visit the CSC office and pay $2 to sign up
+   - `ssh userid@mother-goose.wics.uwaterloo.ca` (for WiCS) - If you don’t have
+     a userid, ask Fatema to create one.
```

Comment:
I would link to the wics-sys mailing list instead of Fatema--this should fall under whoever is currently sysadmin